test/big.js

```diff
@@ -504,6 +504,29 @@ exports.jacobi = function () {
     assert.equal(b1.jacobi(b2), 1);
 };
 
+exports.base2 = function () {
+  assert.equal(bignum('-1234734').toString(2), '-100101101011100101110');
+  assert.equal(bignum('-2').toString(2), '-10');
```

Comment:
Intuitively, I would expect 2's complement.  That said:

1. What should 2's complement look like for arbitrary precision numbers?
2. It's consistent with e.g. `BN_bn2hex()`.